tools/jira_client/jira_client_factory.py

```diff
@@ -9,7 +9,7 @@
 class JiraClientFactory:
     @staticmethod
     def create(jira_access_token, dry_run_stream=None, validate=True):
-        client = jira.JIRA(server=consts.JIRA_SERVER, token_auth=jira_access_token, validate=validate)
+        client = jira.JIRA(server=consts.JIRA_SERVER, token_auth=jira_access_token, validate=validate, max_retries=0)
```

Comment:
So will we have some other indiciation in case of a request failure now?

Reply:
We have retries in some other places, e.g.:
https://github.com/openshift-assisted/assisted-installer-deployment/blob/dff5517458ae0770ffc1ec53ee1e2668aa44cf25/tools/triage/add_triage_signature.py#L2951
https://github.com/openshift-assisted/assisted-installer-deployment/blob/036ed3a555abd3dc33c69f74ca42674a6d526aae/tools/triage/attach_logs_in_jira.py#L45
https://github.com/openshift-assisted/assisted-installer-deployment/blob/f591723c55470fb910ca523093c44ca5eb73d37d/tools/triage/common.py#L13

The reason we "prefer" those retries is that they at least preserve idempotence and don't created redundant objects

As for indications, we could have debug logs for each retry. It's a good idea, I'll create a PR for that as well